webapi/src/test/scala/org/knora/webapi/e2e/v1/ResourcesV1R2RSpec.scala

```diff
@@ -238,6 +238,29 @@ class ResourcesV1R2RSpec extends R2RSpec {
     }
 
 
+    private val search = "/v1/resources?restype_id=http%3A%2F%2Fwww.knora.org%2Fontology%2F0001%2Fanything%23Thing"
+    private val filter = "&searchstr=***"
+
+    private def checkSearch(search: String) = {
```

Comment:
Since this function doesn't check all searches, just a couple of very specific ones, could you give it a more specific name?

Reply:
tried to make it more explicit:
1bbc89c